README.md

```diff
@@ -1,56 +1,37 @@
 # `ayb`
-With `ayb`, all your base can finally belong to you. Move SQL for great justice.
+Ayb is the easiest way to get started writing shareable reproducable database demo apps.  With only 5 commands you can have a sharable database.
 
 [![Build status](https://github.com/marcua/ayb/actions/workflows/tests.yml/badge.svg)](https://github.com/marcua/ayb/actions/workflows/tests.yml)
```

Comment:
I might even move this down further.  Build status is a competence trigger for open source developers, it isn't understood by less technical people.